test/testdata/bootstrappolicy/bootstrap_cluster_roles.yaml

```diff
@@ -4074,7 +4074,6 @@ items:
   metadata:
     annotations:
       authorization.openshift.io/system-only: "true"
-      rbac.authorization.kubernetes.io/autoupdate: "true"
```

Comment:
I was suprised too somewhat, I think what happens is that I am converting back to openshift, and because "true" is the default I just remove it.
I guess what I need to do is keep the annotation even though it is the default.